repo-remotes

```diff
@@ -1,2 +1,6 @@
 lionel-/testthat@webr
-lionel-/cli@webr
+georgestagg/Matrix@webR
+georgestagg/cli@webr
+georgestagg/mgcv@webr
+georgestagg/svglite@webr
+georgestagg/data.table@webr
```

Comment:
I updated cli as well today. It would be nice to maintain webr branches in an organisation to which we both have access.